lib/akamai_rspec/matchers/non_akamai.rb

```diff
@@ -40,11 +40,15 @@ def ssl_client_for_verify_cert(tcp_client, addr, url)
 
 RSpec::Matchers.define :be_verifiably_secure do (verify = OpenSSL::SSL::VERIFY_PEER)
   match do |url|
+    # HTTP is not secure
```

Comment:
I'm not a fan of comments, and would suggest that these ones don't add any real value.